zapcore/write_syncer.go

```diff
@@ -88,6 +88,9 @@ type multiWriteSyncer []WriteSyncer
 // NewMultiWriteSyncer creates a WriteSyncer that duplicates its writes
 // and sync calls, much like io.MultiWriter.
 func NewMultiWriteSyncer(ws ...WriteSyncer) WriteSyncer {
+	if len(ws) == 1 {
+		return ws[0]
+	}
```

Comment:
Added, Currently simply verifying that passing just one argument to NewMultiWriteSyncer simply returns the argument back.